tests/testExam.js

```diff
@@ -119,7 +120,72 @@ describe('Exam assessment', function() {
 
     helperExam.startExam(locals);
 
-    describe('6. save correct answer to question addVectors', function() {
+    describe('6. assessment_instance: set attach files page URL', function() {
```

Comment:
Numbering these seems to require a lot of extraneous diffs when tests are added or removed - is there a reason these were numbered in the past?

Reply:
The numbering was originally because I was annoyed at grepping for the test that was failing, and unique numbers made this easier. I agree the diff noise (and need to edit) is very annoying as well 😄. I guess this is all part of my general dissatisfaction with Jasmine-type test runners (mocha, jest, etc). I wish they would be less clever and just print error messages right when they happen. I also wish they didn’t do their async contortions, which force us into the horrible `locals` hack inside our tests, rather than us being able to simply pass variables around. Ok, enough of me ranting about JS test frameworks. I’m happy to drop the test numbering anytime, but I won’t both removing it now unless you think this is a good time to do it?